scripts/config-gitea.ps1

```
@@ -0,0 +1,134 @@
+    Write-Error "Could not extract token from response."
+    Stop-Job $Job
+    Remove-Job $Job
+    exit 1
+}
+
+Write-Host "Token created successfully."
+
+# Update .env files
+$envFiles = @(".env", "apps/portal/.env")
+
+function Update-EnvVar {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Update-EnvVar' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)